public/usage-examples/graphics/draw_line_on_window/draw_line_on_window-1-simple-oop.cs

```diff
@@ -0,0 +1,30 @@
+using SplashKitSDK;
+
+namespace draw_line_on_window
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Create Window
+            Window start = new Window("draw line on window", 600, 600);

```

Comment:
For the Window name, I suggest using capital 1st letters so that it looks more like a heading. It would also be better if this reflected what the example is showing rather than the function name.

Also, update the Window variable name to something like `window` or `line_window`.